component/authenticator/pom.xml

```diff
@@ -232,5 +237,15 @@
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.governance</groupId>
+            <artifactId>org.wso2.carbon.identity.captcha</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
+            <artifactId>org.wso2.carbon.identity.organization.management.service</artifactId>
+            <version>1.0.42</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why did we need this?

Reply:
This was added as a dependency to resolve the following error that occured when mocking FrameworkServiceDataHolder class in EmailOTPAuthenticatorTest class.
```
[ERROR] java.lang.IllegalStateException: Failed to transform class with name org.wso2.carbon.identity.application.authentication.framework.internal.FrameworkServiceDataHolder. Reason: cannot find org.wso2.carbon.identity.organization.management.service.OrganizationManagementInitialize
```